internal/configs/version1/nginx-plus.tmpl

```diff
@@ -35,7 +35,9 @@ http {
     {{- end}}
 
     {{if .LogFormat -}}
-    log_format  main  '{{.LogFormat}}';
+    log_format  main {{if .LogFormatEscaping}}escape={{ .LogFormatEscaping }} {{end}}{{range $value := .LogFormat -}}
```

Comment:
for readability of the template, could you put the range loop one line below. For example:
```
    log_format  main {{if .LogFormatEscaping}}escape={{ .LogFormatEscaping }} {{end}}
                     {{range $value := .LogFormat -}}
                     {{with $value }}'{{ $value }} '{{end}}
                     {{end}};
```